canonical-web-and-design/www.ubuntu.com
# 5166 implement events and webinars

## Done

- Added upcoming webinars and events to /blog

## QA

- Check out this feature branch
- Run the site using the command `./run`
- View the site locally in your web browser at: [/blog](http://0.0.0.0:8001/blog)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure the "Upcoming events and webinars" section towards the bottom of the page looks and behaves similarly to https://blog.ubuntu.com
- Ensure that when "See more ›" is clicked, the archives page shows posts matching both webinar and event categories


## Issue / Card

Fixes #5166
